### ISSUE TITLE
integration stops wroking

### ISSUE THREAD

**Issue by YvesDC** [dan-r/HomeAssistant-Ohme#71]
The integration works fine, but after a while it no longer gets any data.  It shows an error in the integration page. When trying to reconfigure it shows 'unknown error'.
What solves it is delete the integration, restart and add & configure the integration .
Has happened already a couple of time
I think the issues started after upgrading HomeAssistant

[home-assistant_ohme_2024-04-09T08-31-10.381Z.log](https://github.com/dan-r/HomeAssistant-Ohme/files/14915990/home-assistant_ohme_2024-04-09T08-31-10.381Z.log)

**Comment by dan-r** [comment]
Looks like some network issues with Ohme and some of your other integrations from that log.
When the integration stops working, do you have any issues accessing the internet on other devices?

**Comment by yvesdecloedt** [comment]
Hi Dan,
I happened 2 _3 times (not in the last days). Could have been a network issue, however it did no resolve. The integration did never reconnect. I tried reconfiguring , bu t always got 'unknown error' (and at that moment there were certainly no network issues, and I could connect to Ohme via mobile on wifi)
Only after re-installing and restrating did I get connection

**Comment by yvesdecloedt** [comment]
hi,
yes I am.

I am running this on a Mac running Home assistant in a VM (under UTM).
My Raspberry is used at this tile for Homebridge & node-red.
I plan on adding another raspberry for home assistant.

So it stopped again a couple of days back. I have rebooted everything (-> failed to setup). If I will re install then it will work again (as I did the last couple of times). 
I have not don yet as maybe you would like me to try something first ?
